Makefile

```diff
@@ -48,6 +48,16 @@ update-deps: ## Check pyproject.toml for changes, update the lock file if needed
 run: ## Run the service locally
 	python runner.py
 
+memray-run: ## Run the service locally using memray
+	python -m memray run -o ./memory_profile.bin runner.py
+
+memray-flamegraph: ## Extract flamegraph from memray output
```

Comment:
you can make it as dependency for memray-run, right

Reply:
Changed as suggested.